honeybadgermpc/progs/jubjub.py

```diff
@@ -198,15 +194,24 @@ def __str__(self):
 
         return self
 
-    async def mul(self, n):
+    def mul(self, n):
         if not isinstance(n, int):
             raise Exception("Can't scale a point by something which isn't an int!")
 
         return self
 
-    async def double(self):
+    def double(self):
         return self
 
+    async def equals(self, other):
```

Comment:
why is this async?

Reply:
It seems cleaner than returning a `Future` which will evaluate to `True` or `False`, as it's nonsensical to call either this or `open` without awaiting immediately (or gathering).

Reply:
I think I should have phrased my concern better, there is no `await` in this method and the one below, so why do you even need the `async` keyword?

Reply:
I think the semantics of SharedPoint and SharedIdeal should be the same, having this be async makes the code itself cleaner. Perhaps returning a future is better here.